README.adoc

```diff
@@ -76,6 +76,16 @@ If you run into any issues during the development process, please fill out a
 developer https://support.yubico.com/support/tickets/new[support ticket] and
 our team will be happy to assist you.
 
+=== Support for desktop apps
+WARNING: This is an introductory version and is not intended for production use.
```

Comment:
I would call it an "experimental" version rather than "introductory". Also add a line stating that the API should not be considered stable, and may change with new versions.